readthedocs/doc_builder/environments.py

```diff
@@ -73,7 +73,7 @@ def __init__(
         bin_path=None,
         record_as_success=False,
         demux=False,
-        **kwargs,
+        **kwargs,  # pylint: disable=unused-argument
```

Comment:
This seems like we have linting configured too strongly, if so much of this PR is linting hints... Maybe that's making the code better, but I'm not convinced this check is useful?